kr_mesh_visualization/package.xml

```diff
@@ -1,20 +1,25 @@
-<package format="2">
+<package format="3">
   <name>kr_mesh_visualization</name>
   <version>1.0.0</version>
   <description>kr_mesh_visualization</description>
-
+  
   <maintainer email="kartikmohta@gmail.com">Kartik Mohta</maintainer>
   <maintainer email="michael.f.watterson@gmail.com">Michael Watterson</maintainer>
-
+  <maintainer email="pranavpshah2098@gmail.com">Pranav Shah</maintainer>
   <license>BSD</license>
-
   <author>Kartik Mohta</author>
 
-  <buildtool_depend>catkin</buildtool_depend>
+  <buildtool_depend>ament_cmake</buildtool_depend>
 
-  <depend>nav_msgs</depend>
+  <depend>rclcpp</depend>
   <depend>geometry_msgs</depend>
```

Comment:
If the other messages are not included by default remove them from the package.xml

Reply:
The subscribers for the other message types are just commented out in the code. Wouldn't it be easier if the package if only that piece of code has to uncommented and no changes have to be made to CMakeLists or package.xml?

Reply:
I have a better idea for all this. Can we have a param that decides the type of topic that we are using at runtime?